src/main/content/index.html

```diff
@@ -37,17 +44,17 @@ <h3 id="intro_lighten_up_title">A lightweight open framework for building fast a
                 <img id="cloud_2" class="d-none d-md-block" src="/img/home_cloud_1.png" alt="Cloud 2">
                 <img id="beam" src="/img/home_beam.png" alt="Beam">
                 <img id="ufo" src="/img/home_ufo.png" alt="Ufo">
-                <img id="person_1" src="img/home_person_1.png" alt="Person 1">
-                <img id="coffee" src="img/home_coffee.png" alt="Coffee cup">
-                <img id="shoe" src="img/home_shoe.png" alt="Shoe">
-                <img id="laptop" class="d-none d-md-block" src="img/home_laptop.png" alt="Laptop">
-                <img id="ball" class="d-none d-md-block" src="img/home_ball.png" alt="Ball">
-                <img id="dog" class="d-none d-md-block" src="img/home_dog.png" alt="Dog">
-                <img id="phone" class="d-none d-md-block" src="img/home_phone.png" alt="Phone">
-                <img id="mouse" class="d-none d-md-block" src="img/home_mouse.png" alt="Mouse">
+                <img id="person_1" src="/img/home_person_1.png" alt="Person 1">
+                <img id="coffee" src="/img/home_coffee.png" alt="Coffee cup">
+                <img id="shoe" src="/img/home_shoe.png" alt="Shoe">
+                <img id="laptop" class="d-none d-md-block" src="/img/home_laptop.png" alt="Laptop">
+                <img id="ball" class="d-none d-md-block" src="/img/home_ball.png" alt="Ball">
+                <img id="dog" class="d-none d-md-block" src="/img/home_dog.png" alt="Dog">
+                <img id="phone" class="d-none d-md-block" src="/img/home_phone.png" alt="Phone">
+                <img id="mouse" class="d-none d-md-block" src="/img/home_mouse.png" alt="Mouse">
```

Comment:
Are we making these absolute because the `src` values were broken after we added `openliberty.io/de/...`? I can imagine that the non-english urls were doing `openliberty.io/de/img/...`, which causes the images to be 404.